jobs/samba-integration.yml

```diff
@@ -56,7 +56,7 @@
     - shell: !include-raw: scripts/common/get-node.sh
       # the samba-integration:centos-ci branch contains a copy of bootstrap.sh
       # from the gluster/centosci repo
-    - shell: ./bootstrap.sh samba-integration-centos-ci-tests.sh "ghprbPullId=${ghprbPullId} ghprbTargetBranch=${ghprbTargetBranch}"
+    - shell: ./bootstrap.sh samba-integration-centos-ci-tests.sh "ghprbPullId=${ghprbPullId} ghprbTargetBranch=${ghprbTargetBranch} CENTOS_VERSION=$CENTOS_VERSION"
```

Comment:
If I understand this correctly, this hunk is not actually about changing the default, it is about handing the variable `CENTOS_VERSION` into the script in the first place. Originally the version set in the yaml file did not have any effect at all, and the default version that is set in the script itself was always effective (see https://github.com/gluster/centosci/blob/main/jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh#L7). 

So this hunk should probably be a separate commit to enable handing in the variable from the yaml into the script.

Reply:
Alright.